python/nano/src/bigdl/nano/deps/ipex/ipex_inference_model.py

```diff
@@ -124,33 +129,68 @@ def __init__(self, model: torch.nn.Module, input_sample=None, use_ipex=False, dt
                 with torch.no_grad():
                     with torch.cpu.amp.autocast():
                         if self.jit_method == 'trace':
-                            self.model = torch.jit.trace(self.model, input_sample,
-                                                         check_trace=False,
-                                                         strict=jit_strict)
+                            if compare_version("torch", operator.ge, "2.0"):
```

Comment:
And I think you should define a new function to do jit trace instead of copy paste everywhere

Reply:
Good suggestion, thanks! Have revised to this form.